console/src/view/task.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{input, tasks::Task};
+use std::{cell::RefCell, rc::Rc};
+use tui::{
+    layout,
+    text::{Span, Spans},
+    widgets::{Block, Borders, Paragraph},
+};
+
+pub(crate) struct TaskView {
+    task: Rc<RefCell<Task>>,
+}
+
+impl TaskView {
+    pub(super) fn new(task: Rc<RefCell<Task>>) -> Self {
+        TaskView { task }
+    }
+
+    pub(crate) fn update_input(&mut self, _event: input::Event) {
+        // TODO :D
+    }
+
+    pub(crate) fn render<B: tui::backend::Backend>(
+        &mut self,
+        frame: &mut tui::terminal::Frame<B>,
+        area: layout::Rect,
+    ) {
+        // Rows with the following info:
+        // - Task main attributes
+        // - task metadata
+        // - metrics
+        // - logs?
+
+        let task = &*self.task.borrow();
+        const DUR_PRECISION: usize = 4;
+
+        let attrs = Spans::from(vec![Span::raw("ID: "), Span::raw(task.id_hex())]);
+
+        let metrics = Spans::from(vec![
```

Comment:
other metrics that could be nice to show here include:
 - the number of polls
 - time from when the task was spawned to when it was first polled (which we can compute based on data we already have stored)
 - when the task was spawned, and if it has completed, when it finished